libs/wingsdk/test/target-tf-aws/schedule.test.ts

```diff
@@ -0,0 +1,85 @@
+import * as cloud from "../../src/cloud";
+import * as std from "../../src/std";
+import * as tfaws from "../../src/target-tf-aws";
+import { Testing } from "../../src/testing";
+import { mkdtemp } from "../../src/util";
+import { tfResourcesOf, tfSanitize, treeJsonOf } from "../util";
+
+test("schedule behavior with rate", () => {
+  // GIVEN
+  const app = new tfaws.App({ outdir: mkdtemp() });
+  const fn = Testing.makeHandler(
+    app,
+    "Handler",
+    `async handle(event) { console.log("Received: ", event); }`
+  );
+  const schedule = new cloud.Schedule(app, "Schedule", {
+    rate: std.Duration.fromMinutes(2),
+  });
+  schedule.onTick(fn);
+  const output = app.synth();
+
+  // THEN
+  expect(tfResourcesOf(output)).toEqual([
+    "aws_iam_role", // role for function
+    "aws_iam_role_policy", // policy for role
+    "aws_iam_role_policy_attachment", // execution policy for role
+    "aws_lambda_function", // processor function
+    "aws_s3_bucket", // S3 bucket for code
+    "aws_s3_object", // S3 object for code
+    "aws_scheduler_schedule", // main schedule
+  ]);
+  expect(tfSanitize(output)).toMatchSnapshot();
+  expect(treeJsonOf(app.outdir)).toMatchSnapshot();
```

Comment:
Let's add a check in these tests to ensure the scheduler was created with a rate or cron. 

```typescript
expect(
    cdktf.Testing.toHaveResourceWithProperties(output, "aws_scheduler_schedule", {
      schedule_expression: "rate(2 minutes)",
    })
  ).toEqual(true);
```